src/cdk/a11y/focus-monitor/focus-monitor.ts

```diff
@@ -241,21 +241,22 @@ export class FocusMonitor implements OnDestroy {
       this._windowFocusTimeoutId = setTimeout(() => this._windowFocused = false);
     };
 
+    const eventListenerOptions = supportsPassiveEventListeners() ?
```

Comment:
I think the naming here should indicate that it also includes the `capture` option, otherwise somebody might not notice it if we decided to add another listener that should be passive, but non-capturing.

Reply:
Should be better now. Let me know if you have a better name in mind.

Reply:
The new one looks good.